addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/handler/YeelightHandlerBase.java

```diff
@@ -76,6 +76,7 @@ public void initialize() {
         mDevice.registerStatusChangedListener(this);
         updateStatusHelper(mDevice.getConnectionState());
         DeviceManager.getInstance().startDiscovery(15 * 1000);
+        mDevice.connect();
```

Comment:
Added this to make sure it tries to connect. Should normally getting invoked directly from `mDevice.setAutoconnect(true)`, just added to be sure.